tslint.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Are these rules used in addition to the recommended rule set? I'm looking for indent type (spaces), indent size, and whitespace checks to keep checkins clean..